docs/tutorials/developer-journey/level-5/5.5-SNS-tutorial.md

```diff
@@ -0,0 +1,1097 @@
+# 5.5 Launching an SNS
+
+## Overview
+
+A Service Nervous System (SNS) is an advanced form of a DAO. If you recall, DAOs and the NNS were covered in a previous module, [4.4 NNS and governance](../level-4/4.4-nns-governance.md). To recap, a DAO, or decentralized autonomous organization, is a form of decentralized governance that is controlled by the DAO's stakeholders without a central form of authority. The Internet Computer is managed by a DAO known as the Network Nervous System. 
+
+The underlying architecture of an SNS closely resembles the architecture of the NNS, but instead of being used to control the IC, an SNS is used to control a specific decentralized application that is running on the IC mainnet. Any dapp running on the mainnet can be handed over to an SNS, which results in the dapp being owned and controlled by a community. When an SNS takes control of a dapp, the dapp is then managed in the same way that the Internet Computer is managed by the NNS. When an SNS is launched, the ledger defines a unique token for the SNS that is then used by the community for participation in the SNS's governance. 
```

Comment:
Maybe this is important in the tutorial, but why do we say "mainnet"? I wonder why it is important to call it mainnet, compared to just saying "ICP"?

Reply:
In the dev docs, we use "mainnet" to differentiate between local development and development that is deployed on the IC.